optuna/importance/_mutual_information/_evaluator.py

```diff
@@ -0,0 +1,120 @@
+from copy import deepcopy
+import math
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Tuple
+
+import numpy as np
+
+from optuna.distributions import BaseDistribution
+from optuna.distributions import UniformDistribution
+from optuna.importance._base import BaseImportanceEvaluator
+from optuna.importance._mutual_information._parzen_estimator import _MultivariateParzenEstimator
+from optuna.study import Study
+from optuna.trial import TrialState
+
+
+class MutualInformationImportanceEvaluator(BaseImportanceEvaluator):
+    def __init__(self, n_MC: Optional[int] = None) -> None:
+        self._n_MC = n_MC
+
+    def evaluate(
+        self, study: Study, params: Optional[List[str]] = None, n_MC: Optional[int] = None
```

Comment:
Where do you use the argument of  `n_MC`?

Reply:
I just forgot to delete it. Thank you!